Menu/MenuBuilder.php

```diff
@@ -119,7 +119,7 @@ public function createSidebarMenu()
                     }
 
                     $label = $admin->getLabel();
-                    $options = $admin->generateMenuUrl('list');
+                    $options = $admin->generateMenuUrl('list', $item['route_params'], $item['route_absolute']);
```

Comment:
@greg0ire removed, in the future we can make a Admin `list` menu item already with filters or other `route_params`